### PR TITLE
Add 50% discount promo

### PR DESCRIPTION
**Pull request by mattandrews**
This changes a bunch of stuff to promote the new discount rate:

##### Before:
screen shot 2015-05-21 at 12 10 46
##### After:
screen shot 2015-05-21 at 12 11 02

##### Before:
screen shot 2015-05-21 at 12 11 34
##### After:
screen shot 2015-05-21 at 12 11 43

##### Before:
screen shot 2015-05-21 at 12 11 58
##### After:
screen shot 2015-05-21 at 12 12 06

##### Before:
screen shot 2015-05-21 at 12 12 28
##### After:
screen shot 2015-05-21 at 12 12 37

##### Before:
screen shot 2015-05-21 at 12 12 59
##### After:
screen shot 2015-05-21 at 12 13 05

##### Before:
screen shot 2015-05-21 at 12 13 15
##### After:
screen shot 2015-05-21 at 12 13 21

cc @TesterSpike and @davidrapson

### CODE REVIEW

File: frontend/assets/stylesheets/components/_header-bar.scss [hunk 1]
```diff
@@ -30,6 +30,22 @@
             @include fs-headline(7);
         }
     }
+
+    .header-bar__subheading {
+        @include fs-headline(3);
+        padding: rem($gs-gutter / 2) 0 0;
+
+        @include mq(tablet) {
+            // @include fs-headline(4);
```
Comment (davidrapson) [inline frontend/assets/stylesheets/components/_header-bar.scss]: Comment?

**Comment by davidrapson** [comment]
:+1: LGTM aside from the comment

File: frontend/assets/stylesheets/components/_header-bar.scss (hunk 1 above)
Comment (mattandrews) [inline frontend/assets/stylesheets/components/_header-bar.scss]: Oops. Not having much luck with commented-out commits lately... fixed.

**Comment by mattandrews** [comment]
@davidrapson would you mind looking at the [most recent commit](https://github.com/guardian/membership-frontend/commit/6e340bdf0005da67fb95d4750e4c7e30ac582a3f) with some final changes when you get a sec?

**Comment by davidrapson** [comment]
:+1:

**Comment by TesterSpike** [comment]
:+1:

**Pull request merged**